src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery firstBall = new Ball();
+        Lottery secondBall = new Ball();
+        Lottery thirdBall = new Ball();
```

Comment:
it's enough to have only one instance of Lottery + bad naming, think about a better one Lottery != Ball